pkg/api/v1/types.go

```diff
@@ -268,25 +266,26 @@ func (vl *VirtualMachineList) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
-type VMConditionType string
+type VirtualMachinConditionType string
 
 // These are valid conditions of VMs.
 const (
-	// PodCreated means that the VM request was translated into a Pod which can be scheduled and started by
-	// Kubernetes.
-	PodCreated VMConditionType = "PodCreated"
 	// VMReady means the pod is able to service requests and should be added to the
 	// load balancing pools of all matching services.
-	VMReady VMConditionType = "Ready"
+	VirtualMachineReady VirtualMachinConditionType = "Ready"
+
+	// If there happens any error while trying to synchronize the VM with the Domain,
+	// this is reported as false.
+	VirtualMachineSynchronized VirtualMachinConditionType = "Synchronized"
```

Comment:
We need to start testing for this condition in our functional tests. It shouldn't show up in normal operation.

Reply:
Yes, at least for the negative one, where virt-handler sees issues. I am not sure, if a positive check makes any sense. Maybe changing this to an Error condition might also make sense.